serving-api/src/main/scala/com/stratio/sparkta/serving/api/service/http/PolicyContextHttpService.scala

```diff
@@ -129,6 +138,14 @@ trait PolicyContextHttpService extends BaseHttpService {
     }
   }
 
+  def createOutputs(fragmentActor: ActorRef, outputs: Seq[FragmentElementModel]): Future[Seq[Any]] = {
+    val listFutures = for {
+      output <- outputs
+    } yield fragmentActor ? FragmentActor.Create(output)
```

Comment:
You don't need this future, send message with ! inside one outputs.foreach(output => ...)